modules/api/api-xsjs/src/main/resources/META-INF/dirigible/xsk/web/web.js

```diff
@@ -329,30 +334,30 @@ exports.WebResponse = function (clientResponse) {
     this.status; // from $.net.http
 
     this.followUp = function (followUpObject) {
-      var {
+      let {
```

Comment:
Use `const` but check if those variables are not re-set somewhere in the code

Reply:
Not used. Will be changed to const in next commit coming